src/checkstatus/SSMAutomationExecutionStatusWaiter.py

```diff
@@ -0,0 +1,31 @@
+import botocore
+import constants
+import waiter_acceptor_config
+
+def check_automation_execution_status(automation_execution_id, max_attempts, ssm):
+    acceptor_config = waiter_acceptor_config.SSM_AUTOMATION_EXECUTION_COMPLETED
+    model = botocore.waiter.WaiterModel({
+        "version": 2,
+        "waiters": {
+            "SSMAutomationExecutionStatus": {
+                "delay": constants.DELAY_INTERVAL,
+                "operation": "DescribeAutomationExecutions",
+                "maxAttempts": max_attempts,
+                "acceptors": acceptor_config
+            }
+        }
+    })
+
+    automation_execution_filter = [
+        {
+            'Key': 'ExecutionId',
+            'Values': [
+                automation_execution_id,
+            ]
+        }
+    ]
+    waiter = botocore.waiter.create_waiter_with_client('SSMAutomationExecutionStatus', model, ssm)
+    try:
+        waiter.wait(Filters=automation_execution_filter)
+    except botocore.exceptions.WaiterError as e:
```

Comment:
Is the failed automaiton id present in the exception to debug

Reply:
yes